core/src/main/java/io/aiven/klaw/helpers/db/rdbms/HandleDbRequestsJdbc.java

```diff
@@ -858,6 +858,11 @@ public int findAllComponentsCountForTeam(Integer teamId, int tenantId) {
     return jdbcSelectHelper.findAllComponentsCountForTeam(teamId, tenantId);
   }
 
+  @Override
+  public int findAllComponentsCountForUser(String userId, int tenantId) {
+    return jdbcSelectHelper.findAllComponentsCountForUser(userId, tenantId);
+  }
```

Comment:
Will this find all records including ones that have already been approved or deleted? Do we need to add CREATED into this query?